ivy/functional/ivy/utility.py

```diff
@@ -1,3 +1,4 @@
+# for review
```

Comment:
we should not import `current_backend` explicitly, we should instead simply call `ivy.current_backend`